### DISCUSSION TITLE
What's the `Source Control view` in VSCode?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#268] [Extension Development QnA] [answered]
When I read document [Progress API](https://code.visualstudio.com/api/extension-capabilities/common-capabilities#progress-api)](https://code.visualstudio.com/api/extension-capabilities/common-capabilities)

>In the Source Control view
is it mean those views? 
![image](https://user-images.githubusercontent.com/12727582/203899950-548280d5-fc3e-4314-8f7d-58bcc7592701.png)

**Comment by gjsjohnmurray** [comment, marked as answer]
Yes

**Comment by r3m0t** [comment]
You can find translations here-

https://github.com/microsoft/vscode-loc/blob/main/i18n/vscode-language-pack-zh-hans/translations/main.i18n.json#L10524

https://github.com/microsoft/vscode-loc/blob/main/i18n/vscode-language-pack-zh-hans/translations/main.i18n.json#L8118

https://github.com/microsoft/vscode-loc/blob/main/i18n/vscode-language-pack-zh-hant/translations/main.i18n.json
